packages/webcomponents/tools/confirm-publish.ts

```diff
@@ -0,0 +1,6 @@
+if (!process.env.BYPASS) {
```

Comment:
Maybe change to something more specific, like `BYPASS_BUILDER_PUBLISH`? Not really a big deal, but just in case someone has an env variable that is `BYPASS` for some other reason

Reply:
oh yeah or just like ALLOW_PUBLISH or BLOCK_PUBLISH === 'false' or something